edweiss-app/components/lectures/remotecontrol/abstractRmtCtl.tsx

```diff
@@ -28,24 +37,47 @@ interface AbstractRmtCrlProps {
     handleLeft: () => void;
     handleMic: () => void;
     isRecording: boolean;
+    lang: AvailableLangs;
+    setLang: (lang: AvailableLangs) => void;
+
 }
 
-export const AbstractRmtCrl: React.FC<AbstractRmtCrlProps & LightDarkProps> = ({ handleRight, handleLeft, handleMic, isRecording }) => {
+export const AbstractRmtCrl: React.FC<AbstractRmtCrlProps & LightDarkProps> = ({ handleRight, handleLeft, handleMic, isRecording, lang, setLang }) => {
+
+
+    const modalRefLangSelect = useRef<BottomSheetModal>(null);
+
     return (
         <>
             <RouteHeader disabled title={"Lecture's Slides"} />
 
             <TView borderColor='text' m={'md'} mt={'xl'} mb={'xl'} b={0.5} backgroundColor='base' radius={'lg'}>
-                <TView mt={60} mb={70} justifyContent='center' alignItems='center'>
-                    <TText size={25} mb={'md'}> {t(`showtime:showtime_title`)}</TText>
-                    <TText size={35} >{t(`showtime:rmt_cntl_title`)}</TText>
-                    {isRecording ? (
-                        <TText mt={15} size={15} color='red'> {t(`showtime:recording_start`)} </TText>
-                    ) : (
-                        <TText mt={15} size={15} color='green'> {t(`showtime:tap_to_start_recording`)} </TText>
-                    )}
+
+                {/* STRC text, set language and timer */}
+                <TView mt={17} mb={50} justifyContent='center' alignItems='center'>
+                    <TText size={18} mt={'sm'} mb={'sm'}> {t(`showtime:showtime_title`)}</TText>
+
+                    <TView alignItems='center' flexDirection='row' justifyContent='space-between' mt={20}>
+                        <TTouchableOpacity
+                            backgroundColor='crust'
+                            borderColor='text' p={10} b={1} ml={'md'} radius={1000}
+                            onPress={() => modalRefLangSelect.current?.present()}
+                            testID='strc-lang-button'>
+                            <Icon size={40} name='language-outline' color='text'></Icon>
+                        </TTouchableOpacity>
+
+                        <TView flex={1}></TView>
```

Comment:
I see that removing this has a weird effect on the display, but perhaps it would be better to understand why the justifyContent='space-between' is not working because this solution seems a bit unsecure and confusing

Reply:
That's a great point that you hightlight in here.

Before hand I would like to stress that I have tested many things before getting it done correctly,

The thing is that I wanted to make sure that the two element will be put to the extremity independently of the  components between them. A way of solving this issue is using `anchor`, but this is a bad practice as it implies hard-coded values and phone dependent UI.
This solution, for me is the cleanest, and respond the needs for modularirty and scalibity introducing the minimal amount of technical debt. In fact, if we decide to put another component in between, we could do it just by filling the <TView> with other stuffs, and the 2 buttons will still be separated nice and smoothly.

Do you agree now with my stand point ?

Reply:
Yes I understand